src/components/common/AddLink.tsx

```diff
@@ -0,0 +1,42 @@
+import { makeStyles } from '@mui/styles';
+import { Box, Theme } from '@mui/material';
+import { Icon } from '@terraware/web-components';
+import Link from 'src/components/common/Link';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  addIcon: {
+    fill: theme.palette.TwClrIcnBrand,
+  },
+  disabledIcon: {
+    fill: theme.palette.TwClrBaseGray300,
```

Comment:
base color instead of a semantic one?

Reply:
we're not really using disabled icons, what is this for?

Reply:
@hannah-cai here's the use-case (a completed viability test)
<img width="345" alt="Terraware App 2022-12-09 13-26-18" src="https://user-images.githubusercontent.com/1865174/206798743-7488915e-5c49-420b-8715-8d390f388feb.png">

Reply:
@charlie-tf good catch, i copied it without checking for correctness. will update both

Reply:
removing the color change, just using opacity 0.5 as per our design

Reply:
<img width="335" alt="Terraware App 2022-12-09 13-31-18" src="https://user-images.githubusercontent.com/1865174/206799403-5b6acf63-ef0e-48bb-ac31-82e5a6134951.png">